psi4/psi4Config.cmake.in

```diff
@@ -115,6 +115,11 @@ endif()
 if(NOT TARGET ${PN}::core)
     include("${CMAKE_CURRENT_LIST_DIR}/${PN}Targets.cmake")
     set(CMAKE_CXX_STANDARD @CMAKE_CXX_STANDARD@)
+    if (${CMAKE_MINIMUM_REQUIRED_VERSION} VERSION_LESS 3.15)
+        message(WARNING "Calling program/plugin uses old Python detection because cmake_minimum_required (${CMAKE_MINIMUM_REQUIRED_VERSION}) is <3.15. Contact developer to update.")
```

Comment:
```suggestion
        message(WARNING "Calling program/plugin uses old Python detection because cmake_minimum_required (${CMAKE_MINIMUM_REQUIRED_VERSION}) is <3.15. Contact the program/plugin developer to update.")
```